de.dlr.sc.virsat.model.extension.fdir.test/src/de/dlr/sc/virsat/model/extension/fdir/util/FaultTreeHelperTest.java

```diff
@@ -0,0 +1,101 @@
+/*******************************************************************************
+ * Copyright (c) 2008-2019 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.virsat.model.extension.fdir.util;
+
+import static org.junit.Assert.assertEquals;
+
+import java.io.IOException;
+
+import org.junit.Before;
+import org.junit.Test;
+
+import de.dlr.sc.virsat.model.extension.fdir.test.ATestCase;
+import de.dlr.sc.virsat.model.dvlm.concepts.Concept;
+import de.dlr.sc.virsat.model.extension.fdir.model.Fault;
+import de.dlr.sc.virsat.model.extension.fdir.model.FaultTreeNode;
+import de.dlr.sc.virsat.model.extension.fdir.model.FaultTreeNodeType;
+
+/**
+ * Test class for FaultTreeHelper
+ * @author jord_ad
+ *
+ */
+public class FaultTreeHelperTest extends ATestCase {
+	
+	protected Concept concept;
+	
+	@Before
+	public void setUp() throws Exception {
+		String conceptXmiPluginPath = "de.dlr.sc.virsat.model.extension.fdir/concept/concept.xmi";
+		concept = de.dlr.sc.virsat.concept.unittest.util.ConceptXmiLoader.loadConceptFromPlugin(conceptXmiPluginPath);
+	}
+
+	@Test
+	public void testCopyNode() throws IOException {
+		Fault rootCMSimple = createDFT("/resources/galileo/cm_simple.dft");

```

Comment:
Cant we use a simpler tree than cm_simple? I think if there is a bug its kind of annoying to debug.
Maybe just csp2?